app/models/proof_status.rb

```diff
@@ -1,4 +1,4 @@
-class ProofStatus < ActiveRecord::Base
+class ProofStatus < LocIdBaseModel
```

Comment:
IRRC this class should switch from LocId to slug based routing in this pull request ;)